xml/admin_caasp_advanced_config.xml

```diff
@@ -142,10 +142,10 @@ mountSecret: ceph-user1-secret
 </screen>
    <para>
     If you want the Rook &ceph; agent to require a <literal>mountUser</literal>
-    and <literal>mountSecret</literal> to be set in StorageClasses using Rook,
-    you need to set the environment variable
+    and <literal>mountSecret</literal> to be set in StorageClasses using
+    &rook;, you need to set the environment variable
     <varname>AGENT_MOUNT_SECURITY_MODE</varname> to
-    <literal>Restricted</literal> on the Rook &ceph; operator deployment.
+    <literal>Restricted</literal> on the &rook; &ceph; operator deployment.
```

Comment:
Should the rookceph entity be used here? (Thanks for pointing me to the SES entity repo)
    <literal>Restricted</literal> on the &rookceph; operator deployment.
```

Reply:
Oh, yes! Please!